Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN dnf update -y && \
         php-opcache \
         php-json \
         php-pear \
+        php-apcu \
```

Comment:
Omeka checks for APCU and uses this as a cache in Doctrine, added to help production caching.

Reply:
What are we using APCu to cache? It's pretty old these days.

Reply:
Used for Doctrine cache by Omeka: https://github.com/omeka/omeka-s/blob/b9c9a9790fd5e539fe6225b92fe546d1d872975c/application/src/Service/EntityManagerFactory.php#L54

Reply:
Ah, just the annotation metadata cache.

Reply:
yeh, still no query cache as far as I can tell